### ISSUE TITLE
After trt optimization, BatchNorm2d between conv2d and relu will be fused, but got wrong result.

### ISSUE THREAD

**Issue by ys0232** [NVIDIA/TensorRT#960]
## Description

```

class myModel(nn.Module):
      def __init__(self):
            super(myModel,self).__init__()
            n_feats = 64
           self.conv1 = nn.Conv2d(3, n_feats,3)
           self.bn1 = nn.BatchNorm2d(n_feats)
           self.relu = nn.ReLU(inplace=True)
     def forward(self,x):
           x = self.conv1(x)
           x = self.bn1(x)
           x = self.relu(x)
        return x
```

pytorch1.7.0 will remove the BatchNorm2d, pytorch1.4.0 still keep BatchNorm2d.
trt 7.0 and trt7.1 will fuse conv2d weight with scales from batchNorm2d, and fuse with relu.
after trt optimization, only one layer in the network, and the result of this optimized network is totally different from the original one running on pytorch.

can anyone help to explain that?

## Environment

**TensorRT Version**:  7.1
**GPU Type**: T4
**Nvidia Driver Version**: 418
**CUDA Version**: cuda 11
**CUDNN Version**: 8
**Operating System + Version**: 
**Python Version (if applicable)**: 3.6
**TensorFlow Version (if applicable)**: none
**PyTorch Version (if applicable)**: 1.7.0
**Baremetal or Container (if container which image + tag)**:

**Comment by ys0232** [comment]
@rmccorm4

**Comment by pranavm-nvidia** [comment]
@ys0232 Can you share the ONNX model as well?

**Comment by ys0232** [comment]
`import os
import torch.onnx
import torch.nn as nn
# torch.random.seed()
import numpy as np
torch.manual_seed(1)
torch.cuda.manual_seed_all(1)
class myModel(nn.Module):
    def __init__(self):
        super(myModel,self).__init__()
        n_feats = 64
        self.conv1 = nn.Conv2d(3, n_feats,3)
        self.bn1 = nn.BatchNorm2d(n_feats)
        self.relu = nn.ReLU(inplace=True)

    def forward(self,x):
        x = self.conv1(x)
        x = self.bn1(x)
        x = self.relu(x)

        return x

model = myModel()
img_size=64
data = np.arange(3*img_size*img_size).reshape((1,3,img_size,img_size))
warm_up_tensor = torch.from_numpy(data).type(torch.float32)
torch.onnx.export(model, warm_up_tensor,'batchNorm2d.onnx', opset_version=11, verbose=True, do_constant_folding=False)

res = model(warm_up_tensor)
print(res.flatten(),res.size())`

After running this code, you can get the ONNX file.
@pranavm-nvidia

**Comment by ys0232** [comment]
> ```
> import torch.onnx
> import torch.nn as nn
> # torch.random.seed()
> import numpy as np
> torch.manual_seed(1)
> torch.cuda.manual_seed_all(1)
> class myModel(nn.Module):
>     def __init__(self):
>         super(myModel,self).__init__()
>         n_feats = 64
>         self.conv1 = nn.Conv2d(3, n_feats,3)
>         self.bn1 = nn.BatchNorm2d(n_feats)
>         self.relu = nn.ReLU(inplace=True)
> 
>     def forward(self,x):
>         x = self.conv1(x)
>         x = self.bn1(x)
>         x = self.relu(x)
> 
>         return x
> 
> model = myModel()
> img_size=64
> data = np.arange(3*img_size*img_size).reshape((1,3,img_size,img_size))
> warm_up_tensor = torch.from_numpy(data).type(torch.float32)
> torch.onnx.export(model, warm_up_tensor,'batchNorm2d.onnx', opset_version=11, verbose=True, do_constant_folding=False)
> 
> res = model(warm_up_tensor)
> print(res.flatten(),res.size())
> ```

After running this code, you can get the ONNX file.
@pranavm-nvidia

**Comment by ys0232** [comment]
I have found the mistake, when i set model() to model.eval() on pytorch. Then the result running on pytorch is the same as this one after trt optimization. Thanks for your time~
@pranavm-nvidia
